samples/vue/server/server.js

```diff
@@ -115,6 +115,8 @@ export function renderView(callback, path, data, viewBag) {
           html = assertReplace(html, '<body>', `<body ${meta.teleports.bodyAttrs || ''}>`);
         }
 
+        html = html.replace(new RegExp('phkey', 'g'), 'key');
```

Comment:
Should we use `assertReplace` here for consistency?

Reply:
In case if we will not have placeholders on the page - we will have error, if use `assertReplace`

Reply:
Yep, you're absolutely right.